ui/src/pages/ErrorReport.vue

```diff
@@ -0,0 +1,71 @@
+<template>
```

Comment:
If possible, this would be better handled as a pop-up dialog so that the user doesn't lose the context of where he or she is in the app.